docker/server/nginx/etc/nginx/conf.d/universe.marathon.mesos.conf

```diff
@@ -72,13 +72,25 @@ server {
     types {
       "application/vnd.dcos.universe.repo+json;charset=utf-8;version=v3" json;
     }
+    
+    if ($dcos_release_version = "1.10") {
+      rewrite ^/v3$ /repo-empty-v3.json break;
+    }
+    
+    
     rewrite ^/v3$ /repo-up-to-$dcos_release_version.json break;
   }
 
   location = /v4 {
     types {
       "application/vnd.dcos.universe.repo+json;charset=utf-8;version=v4" json;
     }
+    
+    if ($dcos_release_version != "1.10") {
+      rewrite ^/v4$ /repo-empty-v3.json break; # empty v3 = empty v4
```

Comment:
The correct way to do this is to have the gen-universe.json generate all possible combinations with names `repo-up-to-${dcos-release-version}-${universe-version}.${format}. Where format is either `zip` or `json`. Then just do rewrites to the correct file based on the headers the users give us. It would make the nginx conf files a lot simpler, and we could use the same one in all places.